.github/workflows/auto-trigger-aas-release.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto trigger AAS version bump
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  trigger_aas_release:
+    runs-on: ubuntu-latest
+
+    steps:        
+      - name: Trigger AAS release
+        run: |
+          curl -X POST \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: Bearer ${{ secrets.GH_TOKEN_PIERO }}" \
```

Comment:
Is this correct? `GH_TOKEN_PIERO`? Is it added to the secrets?

Reply:
Yes and Yes.
I can rename but as it's my token, I felt it made sense (oh and yeah, I often use Piero as a nickname.)

Reply:
> I often use Piero as a nickname.
I assumed you'd typo'd your own name 😂